### ISSUE TITLE
Carrot Sword 

### ISSUE THREAD

**Issue by mecsafonso** [sunflower-land/sunflower-land#561]
Hello @adamhannigan !!

I created this issue in order to comunicate that there is a problem while minting the carrot sword:

![image](https://user-images.githubusercontent.com/49355413/163632832-74fe2b76-3471-409f-b772-6d97c8f4114b.png)

I have all the mats needed, farm id: 1149

Hope this gets a fix!

**Comment by brian-man** [comment]
There was a comment I saw by at some point last night / this morning (my time) about Carrot Sword actually consuming 100 SFL rather than 50 SFL.

I took a look at the Carrot Sword items on OpenSea and found two mint transactions from 14hr and 9hr ago.

https://polygonscan.com/tx/0x00c57025c59361d6dbd9349fb2d84d8d29eb4ce481b51c682200e66f8e281d68
Consumes 59.9819 wallet SFL

https://polygonscan.com/tx/0x0299677c0cc4fb7e122c4fe6f159198f71fe62fbf56d41e98dd91d2b55cf2432
Consumes 100 wallet SFL

This seems to support the theory that the actual cost/consumption is higher than the 50 SFL in the UI.
